src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/CommandFactory.shared.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xamarin.CommunityToolkit.ObjectModel;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Helpers
+{
+	public static class CommandFactory
+	{
+		public static Command Create(Action execute, Func<bool> canExecute = null) =>
```

Comment:
Why don't return `ICommand` just like the other `Create` methods that return `IAsyncCommand`?

Reply:
`ICommand` doesn't have `ChangeCanExecute` method while `IAsyncCommand` does have `RaiseCanExecuteChanged`.
If it returned `ICommand` this would hide `ChangeCanExecute` method that is needed to notify of `canExecute` value change.

Reply:
Got it. So, in our samples, should we change all `ICommand` to `Command`?

Reply:
- If ViewModel needs to execute `ChangeCanExecute()` then property must be `Command`, if not, it can be `ICommand`. 
- If property is `ICommand` both `Command` and `AsyncCommand` can be asigned to if

I don't have strong opinion in any direction here.

Reply:
@maxkoshevoi Ok, agreed. Do you mind creating a sample page for that? One that we need to execute the ChangeCanExecute, just to have a reference for the devs?

Reply:
That's just basic Xamarin MVVM staff. This page wouldn't demonstrate anything from XCT. Are you sure it needs to be added?

Reply:
IF you think that it doesn't add any value, it's ok. I was just guessing. TBH Command isn't my main area here.

Reply:
It will have value (who doesn't like samples), but it wouldn't demonstrate anything that could be done only with XCT. `ICommand` interface and `Command` class are standard tools that Xamarin developers are already using. 

TLDR: `ChangeCanExecute` is a method that needs to be called so that Xamarin would rerun `canExecute` function for the given command and update page layout according to result.

Reply:
got it!